docs/Video Streaming for Navigation Apps/Menus/index.md

```diff
@@ -0,0 +1,68 @@
+# Video Streaming Menu
+
```

Comment:
There needs to be an introduction here explaining the general concept of SDL menus. I'm thinking of something like "When building your video-streaming navigation app, you can either create a menu using your own video streaming UI or use the built-in SDL menu system. The SDL menu system allows you to display a menu structure for users to select menu options or submenus. For more information about the SDL menu system, see [link here]. It's recommended to use the built-in SDL menu system in order to have better performance, automatic driver distraction support – such as list limitations and text sizing, and more.

To open the SDL built-in menu from your video streaming UI, see 'Opening the Built-In Menu' below."

Reply:
added

Reply:
added